HW_7/task_34.py

```diff
@@ -0,0 +1,25 @@
+# Задача 34: Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу. 
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке.
+
+# Ввод: 
+# пара-ра-рам рам-пам-папам па-ра-па-дам 
+
+# Вывод:
+# Парам пам-пам
+
+
+def f(word):
+    return sum(1 for i in word if i in 'аеёиоуыэюя')
```

Comment:
Можно иначе
```suggestion
    return sum(i in 'аеёиоуыэюя' for i in word)
```
Подумайте, что я сделал и почему так тоже будет работать

Reply:
предложенный вами код схож с моим, просто вы присваиваете i сразу 'аеёиоуыэюя' без условия if. И получается i в генераторе прогоняется уже по присвоенным 'аеёиоуыэюя'. Если я не путаю, это называется проверка вхождения элемента в генератор?

Reply:
Нет, не совсем так. В i по прежнему передается буква из слова

Reply:
Тогда затрудняюсь ответить. Погуглил информацию по цикл for, но там похожего не нашел🤷‍♂️